src/OpenLoco/Map/TileManager.cpp

```diff
@@ -188,6 +210,92 @@ namespace OpenLoco::Map::TileManager
         return std::make_tuple(height, waterHeight);
     }
 
+    std::tuple<int16_t, int16_t> getHeight(map_pos pos)
```

Comment:
do we need the tuple version in addition to the normal getHeight?

Reply:
What code change are you suggesting?

Reply:
Removing this get height function

Reply:
I thought we wanted to have at least overloads that used the position structures though?